@ndlib/gatsby-theme-marble/src/components/Shared/MarbleItem/ItemLayout/TombstoneMetadata/TombstoneField/index.js

```diff
@@ -11,7 +11,7 @@ const TombstoneField = ({ field, sxStyle, searchField, uriValue }) => {
       {
         field.map((values, i) => {
           return values.value.map((value, j) => {
-            const uri = uriValue != null ? uriValue : encodeURI(value)
+            const uri = uriValue != null ? uriValue : encodeURI(value).replace(/&/g, '%26')
```

Comment:
Switching `encodeURI` to `encodeURIComponent` seems like a better solution unless doing so would cause problems with other "reserved" characters.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURI#examples
@jonhartzler thoughts?

Reply:
yeah lets give that a shot